src/utils/filmUrlSettings.ts

```diff
@@ -0,0 +1,10 @@
+const apiKey = '?api_key=a912f6cd4d0573f728f2dba5b8aa1f6c';
```

Comment:
just as a suggestion for the future, it would be better to add API keys as constants and name variables using uppercase
```suggestion
const API_KEY = 'a912f6cd4d0573f728f2dba5b8aa1f6c';
```